docs/scripts/generate_api_docs.py

```diff
@@ -76,6 +76,19 @@
     ],
 }
 
+def is_documented_method(obj):
+    is_routine = inspect.isroutine(obj)
+    if not is_routine:
+        return False
+    name = obj.__name__
+    # Exclude methods defined in external libraries
+    module = getattr(obj, "__module__", "")
```

Comment:
I don't think this is required, during traversal we only look at things defined inside dspy/